src/parsers/css-value-tokenizer.ts

```diff
@@ -0,0 +1,93 @@
+import { tokenize } from "../core";
+import {
+  isStringDelimiter,
+  isWhitespace,
+  createToken,
+  getText,
+} from "../helpers";
+import type { Token, Descriptors } from "../types";
+
+type Delimiters = "(" | ")" | ",";
+
+export type CSSCodeToken = Token<Descriptors | Delimiters>;
+export type CSSCodeAst = StringNode | MethodCall | TextNode;
+export interface ASTNode<Types = Descriptors> extends Token<Types> {
+  beforeText: string;
+  afterText?: string;
+}
+
+export interface MethodCall extends ASTNode<"call"> {
+  name: string;
+  args: CSSCodeAst[];
+}
+export interface StringNode extends ASTNode<"string"> {}
+export interface TextNode extends ASTNode<"text"> {}
+
+export function createCssValueAST(source: string): CSSCodeAst[] {
+  return getDeclValueTokens(
+    tokenize<CSSCodeToken>(source, {
+      isDelimiter,
+      isStringDelimiter,
+      isWhitespace,
+      shouldAddToken,
+      createToken,
+    })
+  ).ast;
+}
+
+const isDelimiter = (char: string) =>
+  char === "(" || char === ")" || char === ",";
+
+const shouldAddToken = () => true;
+
+function getDeclValueTokens(
```

Comment:
I'm not sure about the afterText  beforeText. What are they for? why not separate token?